### PR TITLE
Remove old Python 2 import in cparser

### PR DESCRIPTION
**Pull request by saimn**

### CODE REVIEW

File: astropy/io/ascii/cparser.pyx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import os
 import math
 import multiprocessing
 import mmap
+import queue as Queue
```
Comment (bsipocz) [inline astropy/io/ascii/cparser.pyx]: nitpicking, but there is reason to call it `Queue` any more?

Comment (saimn) [reply astropy/io/ascii/cparser.pyx]: Yes, `queue` is used as variable name in the code. I could rename these but it seems easier to keep the import like this.

Comment (bsipocz) [reply astropy/io/ascii/cparser.pyx]: ah, ok, fair enough, I didn't look into the code.

**Comment by bsipocz** [comment]
Thanks @saimn!

**Pull request merged**